addons/ai/XEH_preInit.sqf

```diff
@@ -6,4 +6,11 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+#include "initSettings.sqf"
+
+if (isServer) then {
+    GVAR(assignNVGpfh) = -1;
+    GVAR(assignNVGstate) = false;
+};
```

Comment:
Could the CBA setting be set and run before it gets to here? If yes, `GVAR(assignNVGpfh)` would be overwritten.

Reply:
I don't think so, this is all unscheduled, so adding the setting and initializing this variable should happen right one after another. And as far as I know, settings initialize in `postInit`.